web/package.json

```diff
@@ -21,7 +21,7 @@
     "@redwoodjs/forms": "4.4.3",
     "@redwoodjs/router": "4.4.3",
     "@redwoodjs/web": "4.4.3",
-    "@x-image-privacy/wordcloud": "github:x-image-privacy/word-cloud#main",
+    "@x-image-privacy/wordcloud": "https://github.com/x-image-privacy/word-cloud#main",
```

Comment:
Curious why you needed this change ?

Reply:
I had this error:
```
warning Error: REMOTE HOST IDENTIFICATION HAS CHANGED!         
Fatal Error: Could not read from remote repository.
```

And I find a solution here:

https://stackoverflow.com/questions/13509293/git-fatal-could-not-read-from-remote-repository

Reply:
I think this issue was most-likely caused by: https://github.blog/2023-03-23-we-updated-our-rsa-ssh-host-key/ 
GitHub needed to rotate their RSA keys, check if following the code in the article you can get it running. Otherwise using HTTPS is fine 🤷